src/IntegralArrays.jl

```diff
@@ -1,5 +1,78 @@
 module IntegralArrays
 
 # package code goes here
+using Images, ClosedIntervals
```

Comment:
Just looked over the source code. Doesn't look that friendly to writing tight loops:
- the order-switching means there's a branch on construction
- the testing for empty intervals is a lot less clean that the way UnitRanges work

I wonder if we should just start a tiny package of our own.